data/test/scenarios/wml_tests/UnitsWML/zero_experience.cfg

```diff
@@ -0,0 +1,77 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [unit] max_experience=0
+##
+# Actions:
+# Change Alice's and Bob's max_experience to 0. Delete Bob's 'Orcish Warrior' advancement but not Alice's. Harm bob with 1 hitpoint to be sure that he have some damage. They then fight each other.
```

Comment:
For the first sentence, I think "Try to set Alice's and Bob's max_experience to 0, check that the engine applies a minimum of 1." would be clearer. It's also good to add asserts in the middle of the test, to check that's true both before and after the combat.

BTW, Alice is male (just because Elvish Archers are by default).